manifests/params.pp

```diff
@@ -23,7 +23,7 @@
         'user'    => 'haproxy',
         'group'   => 'haproxy',
         'daemon'  => '',
-        'stats'   => 'socket /var/lib/haproxy/stats',
+        'stats'   => 'socket /var/run/haproxy.sock mode 600 level admin',
```

Comment:
is it possible that the path depends on the operating system or on the haproxy version?